src/collective/cover/static/js/main.js

```diff
@@ -1,8 +1,51 @@
 $(function() {
+  // override jquery portlet event
+  // http://stackoverflow.com/a/14063574/2116850
+  function refreshPortlet(hash, _options){
+    var options = {
+        data: {},
+        success: function(){},
+        error: function(){},
+        ajaxOptions: {}};
+    $.extend(options, _options);
+    options.data.portlethash = hash;
+    ajaxOptions = options.ajaxOptions;
+    ajaxOptions.url = $('base').attr('href') + '/@@render-portlet';
+    ajaxOptions.success = function(data){
+      var container = $('[data-portlethash="' + hash + '"]');
+      var portlet = $(data);
+      container.html(portlet);
+      options.success(data, portlet);
+    }
+    ajaxOptions.error = function(){
+      options.error();
+    }
+    ajaxOptions.data = options.data;
+    $.ajax(ajaxOptions);
+  }
+  $('body').undelegate('#calendar-next, #calendar-previous', 'click')
+           .delegate(
+             '.portletWrapper #calendar-next, ' +
+             '.portletWrapper #calendar-previous',
```

Comment:
@hvelarde I have two options here.. or I do this way, or I have to customize 2 templates of both portlet.. you choose.